### PR TITLE
Diagnostic that can be run in context of M365 Administrator

### PR DESCRIPTION
**Pull request by bpeterse03**
I would like to provide this link as a button like what is seen in this article: https://docs.microsoft.com/en-us/sharepoint/troubleshoot/sharing-and-permissions/access-denied-due-to-org-policies 

This diagnostic is intended to identify site/page performance issues in the context as a Microsoft 365 Administrator. It will bring the user to the MAC portal and all they need to do is click Run.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/diagnosing-performance-issues-with-sharepoint-online.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ There are three different ways that you can identify that a page on a SharePoint
 
 - SharePoint Online response header metrics
 
-This topic describes how to use each of these methods to diagnose performance issues. Once you've figured out the cause of the problem, you can work toward a solution using the articles about improving SharePoint performance that you can find on https://aka.ms/tune.
+This topic describes how to use each of these methods to diagnose performance issues. Once you've figured out the cause of the problem, you can work toward a solution using the articles about improving SharePoint performance that you can find on https://aka.ms/tune. Alternatively, if you are an Administrator, and you want to quickly try to diagnose the issue, [run this diagnostic tool] (https://aka.ms/PillarSiteandPagePerf).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/diagnosing-performance-issues-with-sharepoint-online.md]:
```suggestion
This topic describes how to use each of these methods to diagnose performance issues. Once you've figured out the cause of the problem, you can work toward a solution using the articles about improving SharePoint performance that you can find on https://aka.ms/tune. Alternatively, if you are an administrator and you want to diagnose the issue quickly, [run this diagnostic tool](https://aka.ms/PillarSiteandPagePerf). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @petey121175, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@petey121175 Please commit the suggested changes.

**Comment by bpeterse03** [comment]
@yogkumgit @JohanFreelancer9 thank you and sorry for the delay.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**